pyproject.toml

```diff
@@ -23,3 +58,22 @@ lint.extend-select = [
     "Q003",
     "RET",
 ]
+
+[dependency-groups]
+dev = [
+    "aws-sam-cli==1.131.0",
+    "pytest==8.3.3",
+    "pytest-cov==6.0.0",
+    "pytest-django==4.9.0",
+    "pytest-playwright==0.3.0",
+    "aloe-django==0.2.0",
+    "moto==5.0.21",
+    "django-debug-toolbar==5.0.0a0",
+    "ruff>=0.8.1",
+]
+
+deploy = [
+    "aws-sam-cli==1.131.0",
```

Comment:
Does this put us back to trying to resolve SAM's entire massive package tree with our own application? i.e: could SAM's really specific pins prevent us upgrading other stuff?

Reply:
Good point, I'll move to the CircleCI installed image.

Reply:
This was another one you had doubled up.
In https://github.com/DemocracyClub/electionleaflets/pull/287/commits/ac70077cf9ddc5cd20c557b66e1048d7b87c3613 you've removed it from the `dev` requirements but left it in `deploy` so all the bumf it pulls in is still in the tree. It wants removing in both places